src/components/TodoForm.js

```diff
@@ -30,21 +31,31 @@ class TodoForm extends React.Component {
 
   render() {
     return (
-      <>
-        <form onSubmit={this.handleSubmit}>
-          <input
+      <CardDeck>
+      {/* Q: How did I break the Card width already??? tried adding  */}
+      <Card sm="6">
```

Comment:
sm="6" is insufficient to set width, unless used specifically with col/row. Reactstrap docs doesn't show it, but a width must be independently defined on a card, there isn't a "default" card width.